libvast/vast/type.hpp

```diff
@@ -824,6 +824,12 @@ bool compatible(const type& lhs, relational_operator op, const data& rhs);
 /// @relates type data
 bool compatible(const data& lhs, relational_operator op, const type& rhs);
 
+/// Checks whether a type is a superset of another.
```

Comment:
Please explain how you define "superset".

Reply:
Also, I would flip the argument and call it `is_subset`, which is the more common way to look at a type: one type is contained (= subset) within another.